english/challenge-marshmellow-to-rock-off/castle-route/narnia.md

```diff
@@ -21,7 +21,13 @@ As you ponder this, a shadowy figure leaps out from behind you with an intent to
 ambush by sheer luck, you unsheathe your new weapon, ready to take your stand against this madness.. 
 and suddenly, your rusty sword starts glowing as bright as your torch. The shadowy fiend's eyes burn with 
 disgust under its dark hood as it scurries to the stairs and vanishes in the darkness. As you examine some
-cryptic language written on the hilt of the sword, you need to decide.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.